TestScripts/setenv-android.sh

```diff
@@ -355,7 +371,7 @@ fi
 
 # Error checking
 if [ ! -e "${ANDROID_TOOLCHAIN}/$LD" ]; then
-    echo "ERROR: Failed to find Android ld. Please edit this script."
+    echo "ERROR: Failed to find Android ld. Please edit this script. When using NDK 22 or hihger make sure to set ANDROID_LD! (bfd, gold)"
```

Comment:
Typo (hihger -> higher)

Reply:
Thanks for the input. -> fixed!

Reply:
Thanks everyone. This fell off my radar. I want to look at this over the weekend.